### ISSUE TITLE
"query[type=43]" is displayed in web interface instead of "query[DS]"

### ISSUE THREAD

**Issue by Morlok8k** [pi-hole/web#54]
##### Expected Behaviour:

The mouse tooltip in Query Types graph should show _just_ the query type.  A, AAAA, DNSKEY, SRV, etc.  Here is a list: https://en.wikipedia.org/wiki/List_of_DNS_record_types

By this, I mean, the text should say "AAAA", not "query[AAAA]".  Also, Type 43 (DS) should show as just "DS", not "query[type=43]"

So, the text "query[" and "]" should be stripped out.  Any "type=nn" should lookup a table and replace the "type=nn" text with the actual Query type.  I've only seen "type=43", but it could apply to other types.
##### Actual Behaviour:

A, AAAA, DNSKEY, and SRV all show up as query[A], query[AAAA], query[DNSKEY], and query[SRV] respectively.  DS shows as "query[type=43]".

Granted, "query[type=43]" is technically a dnsmasq bug, as my log from `/var/log/pihole.log` shows: (as compared to an A record)

```
Mar  6 00:01:09 dnsmasq[2468]: query[A] {website1}.com from 192.168.1.1
Mar  6 00:01:09 dnsmasq[2468]: forwarded {website1}.com to 8.8.8.8  
Mar  6 00:01:09 dnsmasq[2468]: query[type=43] {website2}.com from 192.168.1.1
Mar  6 00:01:09 dnsmasq[2468]: forwarded {website2}.com to 8.8.8.8  
```

(websites and addresses have been changed)

Edit:  also, in the Query Log page, under the Type column, it just shows "type=43".  at least here, the "query[" and "]" text is already stripped out.
##### Steps to reproduce this issue:

It's always there.

**Comment by Mcat12** [comment]
Have you updated to the latest version?

**Comment by PromoFaux** [comment]
@Mcat12 

I think he's talking about this bit:
![image](https://cloud.githubusercontent.com/assets/1998970/13572871/869452ca-e474-11e5-8600-61755913eb91.png)

**Comment by Mcat12** [comment]
Ah, yes that should be updated.

**Comment by PromoFaux** [comment]
Fixed in above commit:

![image](https://cloud.githubusercontent.com/assets/1998970/13606950/04003e50-e546-11e5-8b32-fd66e595adcb.png)

**Comment by Mcat12** [comment]
We also want to handle the type=# to fully fix this.

**Comment by Mcat12** [comment]
If that is a reproducible issue

**Comment by PromoFaux** [comment]
43 = DS, right? Does dnsmasq return any other types as numbers instead of their name?

**Comment by Morlok8k** [comment]
so, i upgraded from raspbian wheezy to jessie, and now the type=43 issue is gone, and dnsmasq reports it as DS.

**Comment by Mcat12** [comment]
It may be worth it to keep wheezy users in mind anyways.

**Comment by Mcat12** [comment]
Fixed with 9b8c92e
